src/openms_gui/source/VISUAL/SpectrumWidget.cpp

```diff
@@ -354,4 +356,12 @@ namespace OpenMS
     event->acceptProposedAction();
   }
 
+  void SpectrumWidget::paintEvent(QPaintEvent * /*event*/)
+  {
+    QStyleOption opt;
+    opt.init(this);
+    QPainter p(this);
+    style()->drawPrimitive(QStyle::PE_Widget, &opt, &p, this);
```

Comment:
add comment on what this does?

Reply:
I don't know what it does. The docu said I need it.